### ISSUE TITLE
[Improvement] To improve ergonomics, is it possible to place the [Expand/Collapse] to the left in the test output log section ? or to improve the display when showing all details ?

### ISSUE THREAD

**Issue by harmin-parra** [pytest-dev/pytest-html#767]
Is it possible to place the Expand/Collapse to the left in the test output log section ?

The reason for that is that when the test outputs very long strings (via print or logging), I need to scroll all the way to the right to click the [Expand/Collapse] button.

This happens when I do the following:

- Click on [show all details]
- Expand a test detail

Display is OK if I do:

- Click on [hide all details]
- Expand a test detail
![Screenshot from 2023-11-07 13-23-11](https://github.com/pytest-dev/pytest-html/assets/16239820/81f74fe9-f62a-4b5e-a662-6710ab694d8b)

**Comment by BeyondEvil** [comment]
> I think the correct fix is rather to not allow the log to expand further to the right than the viewport.

Can you help with this @drRedflint ?

**Comment by gscelsi** [comment]
Yes please, I have exactly the same problem and what makes things worse is that there is no hot-key associated to the expand button, or at least non I could find (using Firefox 120.0).
